requirements.txt

```
@@ -40,4 +41,5 @@ rq==1.5.2
 schwifty==2020.9.0
 sentry-sdk==0.18.0
 simplejson==3.17.2
+spectree==0.3.7
```

```suggestion
spectree==0.3.8
```

Une 0.3.8 est sortie lundi ! https://pypi.org/project/spectree/0.3.8/